pkg/bpf/coll_windows.go

```diff
@@ -9,24 +9,32 @@ import (
 	"github.com/cilium/ebpf"
 )
 
-type CollectionStore struct {
-	collMap map[string]*ebpf.Collection
-}
-
 var (
-	store          CollectionStore
+	store          map[string]*ebpf.Collection
+	pathStore      map[string]*ebpf.Collection
```

Comment:
these variables could mimic the GetXXX functions, like:
```
  collectionByName map[string]*ebpf.Collection
  collectionByPath map[string]*ebpf.Collection

Reply:
Done